qcengine/programs/tests/test_molpro.py

```diff
@@ -27,20 +28,38 @@ def test_molpro_output_parser(test_case):
     output_ref = qcel.models.Result.parse_raw(data["output.json"]).dict()
     output_ref.pop("provenance", None)
 
-    # TODO add `skip` to compare_recusive
+    # TODO add `skip` to compare_recursive
     check = compare_recursive(output_ref, output)
     assert check, check
 
 
 @pytest.mark.parametrize('test_case', molpro_info.list_test_cases())
 def test_molpro_input_formatter(test_case):
 
-    # Get output file data
+    # Get input file data
     data = molpro_info.get_test_data(test_case)
     inp = qcel.models.ResultInput.parse_raw(data["input.json"])
 
-    # Just test that it runs for now
-    # TODO add actual comparison
+    # TODO add actual comparison of generated input file
     input_file = qcng.get_program('molpro').build_input(inp, qcng.get_config())
-    #print(input_file['infiles']['dispatch.mol'])
-    assert input_file.keys() >= {"commands", "infiles"}
\ No newline at end of file
+    assert input_file.keys() >= {"commands", "infiles"}
+
+
+@using_molpro
+@pytest.mark.parametrize('test_case', molpro_info.list_test_cases())
+def test_molpro_executor(test_case):
+    # Get input file data
+    data = molpro_info.get_test_data(test_case)
+    inp = qcel.models.ResultInput.parse_raw(data["input.json"])
+
+    # Run Molpro
+    result = qcng.get_program('molpro').compute(inp, qcng.get_config(local_options={"ncores": 4}))
```

Comment:
You can now call this via `qcng.compute(inp, 'molpro', local_options=config)`, no need to dive into the data.

Reply:
I've updated the command. In this case I'm specifying the number of cores because it isn't necessarily advantageous for the Molpro testjobs to run with all available cores. In some cases it can prevent them from passing if they are run on a machine with lots of cores and a small amount of memory.

Reply:
Also as a side note, it looks like the `local_options` in `qcng.compute(inp, 'molpro', local_options=config)` expects a dictionary of type `Dict[str, str]`, but at least for the `ncores` option a `Dict[str, int]` needs to be specified. Should this be updated to `Dict[str, any]` or something similar or am I missing something?

Reply:
Whoops, yes should be `Dict[str, Any]`. Note that you will need to import Any from typing.